presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestBinaryStatisticsBuilder.java

```diff
@@ -122,4 +112,11 @@ private void assertBinaryStatistics(ColumnStatistics columnStatistics, int expec
             assertEquals(columnStatistics.getNumberOfValues(), 0);
         }
     }
+
+    public static void addValue(BinaryStatisticsBuilder binaryStatisticsBuilder, Slice slice)
+    {
+        int[] offsets = new int[] {0, slice.length()};
```

Comment:
This won't work with nulls, right?

Reply:
OrcWriter for statistics does not track nulls. It only tracks non nulls explicitly so this is fine.